$NetBSD: patch-ab,v 1.1.1.1 2008/08/19 20:32:12 bjs Exp $

Remove *_DISABLE_DEPRECATED--how will the package know what is 
deprecated and what isn't in the future?

Use LDADD as appropriate for libtool.

--- src/Makefile.am.orig	2007-11-18 08:14:07.000000000 -0500
+++ src/Makefile.am
@@ -53,15 +53,10 @@ jack_rack_CFLAGS = \
 	$(XML_CFLAGS) \
 	-DPKGDATADIR=\"$(pkgdatadir)\" \
 	-DPIXMAPDIR=\"$(pixmapdir)\" \
-	-DDTDDIR=\"$(dtddir)\" \
-	-DG_DISABLE_DEPRECATED=1 \
-	-DGTK_DISABLE_DEPRECATED=1 \
-	-DGDK_DISABLE_DEPRECATED=1 \
-	-DGDK_PIXBUF_DISABLE_DEPRECATED=1 \
-	-DGNOME_DISABLE_DEPRECATED=1
+	-DDTDDIR=\"$(dtddir)\" 
 
 
-jack_rack_LDFLAGS = \
+jack_rack_LDADD = \
 	$(JACK_LIBS) \
 	$(GTK_LIBS) \
 	$(GNOMEUI_LIBS) \
